integrationtest/wildfly/README.md

```diff
@@ -0,0 +1,8 @@
+## Explanation of testing profiles
+### Obtaining WildFly distribution
+- The default is that the TS obtains WildFly from Maven and prepares it automatically.
+- To test against WildFly distributions which you prepared yourself, just pass the `-DjbossHome.node1` and `-DjbossHome.node2` properties containing paths to the distributions
+
+### Testing included modules vs. preparing modules 
+- The default mode is that the TS unzips Hibernate Search and required libraries as WildFly modules into the distributions
+- If you want to test against the modules included in the official WildFly distributions, pass the property `-DuseBuiltinModules`
```

Comment:
I see your point, I'm personally just not sure if it's worth the trouble  because it makes the properties quite long, so it's more work for the user to write that when running the TS from command line. But I don't really care, if that's how you guys prefer it, I can change it. What does Sanne think?